<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListView</class>
 <widget class="QWidget" name="ListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QListView" name="listView"/>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="listView_2"/>
   </item>
   <item row="1" column="1">
    <widget class="QListView" name="listView_3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
